build.sbt

```diff
@@ -72,5 +72,4 @@ lazy val docs = project
     docusaurusCreateSite := docusaurusCreateSite.dependsOn(unidoc in Compile).value,
     docusaurusPublishGhpages := docusaurusPublishGhpages.dependsOn(unidoc in Compile).value
   )
-  .dependsOn(root)
```

Comment:
Docs should depend on something. Root project is an aggregator, that's why i suggested not depending on it, but i believe you have flowJVM project (see how actors are set).